score/torchbench_0.0.yaml

```diff
@@ -0,0 +1,227 @@
+benchmarks:
+  test_eval[BERT_pytorch-cpu-eager]:
+    norm: 0.056674222277769734
+    weight: 0.006944444444444444
+  test_eval[BERT_pytorch-cpu-jit]:
+    norm: 0.04845556728573034
+    weight: 0.006944444444444444
+  test_eval[BERT_pytorch-cuda-eager]:
+    norm: 0.026858860157876495
+    weight: 0.006944444444444444
+  test_eval[BERT_pytorch-cuda-jit]:
+    norm: 0.017104738694909322
+    weight: 0.006944444444444444
+  test_eval[LearningToPaint-cpu-eager]:
+    norm: 0.015391482121220757
+    weight: 0.020833333333333332
+  test_eval[LearningToPaint-cpu-jit]:
+    norm: 0.01744512900000531
+    weight: 0.020833333333333332
+  test_eval[LearningToPaint-cuda-eager]:
+    norm: 0.004052860649187367
+    weight: 0.020833333333333332
+  test_eval[LearningToPaint-cuda-jit]:
+    norm: 0.004101277514284299
+    weight: 0.020833333333333332
+  test_eval[Super_SloMo-cuda-eager]:
+    norm: 0.5727992373999768
+    weight: 0.006944444444444444
+  test_eval[Super_SloMo-cuda-jit]:
+    norm: 0.5758905388000131
+    weight: 0.006944444444444444
+  test_eval[attention_is_all_you_nee...-cpu-eager]:
+    norm: 0.8630710880000152
+    weight: 0.006944444444444444
+  test_eval[attention_is_all_you_nee...-cpu-jit]:
+    norm: 0.8489351521999652
+    weight: 0.006944444444444444
+  test_eval[attention_is_all_you_nee...-cuda-eager]:
+    norm: 0.18483205416672868
+    weight: 0.006944444444444444
+  test_eval[attention_is_all_you_nee...-cuda-jit]:
```

Comment:
Is this input data? Should it be using these abbreviated names?

Reply:
well, these are actually the names we are storing in the benchmark .json.  we may want to change that.